test/modules/MetricsValidator/src/tests/ValidateDocumentedMetrics.cs

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft. All rights reserved.
 
-namespace MetricsValidator
+namespace MetricsValidator.Tests
```

Comment:
Usually in our codebase child namespaces map to folders (i.e., `test/modules/MetricsValidator/src/tests`)...

Reply:
Ah, I see you already have that folder, so this file would normally go into that folder if it's in the corresponding namespace

Reply:
It already is?

Reply:
Oh, should I have the src in the namespace? All the other test modules ignore the src folder for their namespace.

Reply:
No you don't need src in the namespace

Reply:
And yes, this source file is already in the tests/ subfolder. No idea what I was thinking...